doc/manual/manual.tex

```diff
@@ -4602,7 +4603,7 @@ \subsubsection{A note on the syntax of formulas in input files}
 case is this snippet of code discussed in
 Section~\ref{sec:cookbooks-simple-box-3d}:
 %
-\lstinputlisting[language=prmfile]{cookbooks/convection_box_3d/doc/initial.part.prm.out}
+\lstinputlisting[language=prmfile]{../../cookbooks/convection_box_3d/doc/initial.part.prm.out}
```

Comment:
Why is this necessary? For the other includes you do not need it.

Reply:
This is the main `manual.tex` file, and we're referencing a `.prm` file that is in one of the cookbooks. Previously, we went through the symlinks created in a previous PR, but those don't work in the cmake setup I'm building. So I used the true relative path within the repository.